db/cache_test.go

```diff
@@ -0,0 +1,173 @@
+package db
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/pokt-network/poktroll/pkg/polylog/polyzero"
+	"github.com/stretchr/testify/require"
+	gomock "go.uber.org/mock/gomock"
+
+	"github.com/buildwithgrove/path/user"
+)
+
+func Test_GetGatewayEndpoint(t *testing.T) {
+	tests := []struct {
+		name       string
+		endpointID user.EndpointID
+		mockReturn map[user.EndpointID]user.GatewayEndpoint
+		expected   user.GatewayEndpoint
+		found      bool
```

Comment:
Reoder the params too - I'm aligning it with the order in which you're validating git.

```suggestion

		expectedEndpointFound      bool
		expectedGatewayEndpoint   user.GatewayEndpoint
```